### PR TITLE
Referencing the shared library and refactoring

### PR DESCRIPTION
**Pull request by guypritchard**

### CODE REVIEW

File: src/MockClassifier.Api/Services/Dmr/DmrService.cs [hunk 1]
```diff
@@ -11,73 +11,82 @@ namespace MockClassifier.Api.Services.Dmr
...
             }
+
+            byte[] bytes = Encoding.UTF8.GetBytes(content);
+            var base64 = Convert.ToBase64String(bytes);
+            return base64;
         }
 
-        private static HttpRequestMessage CreateRequestMessage(DmrRequest request, StringContent content)
+        private static HttpRequestMessage CreateRequestMessage(DmrRequest request)
         {
```
Comment (yahya130) [inline src/MockClassifier.Api/Services/Dmr/DmrService.cs:71]: Would it be useful doing a null check here on the DmrRequest object's field 'request.payload'? that way we know we would not be sending an empty payload? forgive me if I have missed something here whilst making this point

File: src/MockClassifier.Api/Services/Dmr/DmrService.cs [hunk 2]
```diff
@@ -11,73 +11,82 @@ namespace MockClassifier.Api.Services.Dmr
...
+                throw new ArgumentNullException(nameof(content));
             }
+
+            byte[] bytes = Encoding.UTF8.GetBytes(content);
+            var base64 = Convert.ToBase64String(bytes);
+            return base64;
         }
 
-        private static HttpRequestMessage CreateRequestMessage(DmrRequest request, StringContent content)
+        private static HttpRequestMessage CreateRequestMessage(DmrRequest request)
```
Comment (yahya130) [inline src/MockClassifier.Api/Services/Dmr/DmrService.cs:70]: Would it be useful doing a null check here on the DmrRequest object's field 'request.payload'? that way we know we would not be sending an empty payload? forgive me if I have missed something here whilst making this point

**Review by yahya130 — APPROVED** [review]

File: src/MockClassifier.Api/Controllers/MessagesController.cs [hunk 3]
```diff
@@ -41,19 +47,21 @@ public async Task<IActionResult> Post()
             {
                 return BadRequest(ModelState);
             }
+
+            Console.WriteLine(input);
```
[the remark this answers is not in the document]
Comment (guypritchard) [reply src/MockClassifier.Api/Controllers/MessagesController.cs:51]: Yeah - debugging, I'll take that out before I publish this review.

File: src/MockClassifier.Api/Services/Dmr/DmrService.cs (hunk 2 above)
Comment (guypritchard) [reply src/MockClassifier.Api/Services/Dmr/DmrService.cs]: This is gone now!

**Review by tonistuulNG — APPROVED** [review]

**Review by joonasroosalung — APPROVED** [review]

**Pull request merged**